SimFastTiming/Configuration/python/SimFastTiming_EventContent_cff.py

```diff
@@ -20,8 +20,17 @@
     outputCommands = cms.untracked.vstring()
 )
 
-_phase2_timing_extraCommands = [ 'keep *_mix_FTLBarrel_*','keep *_mix_FTLEndcap_*','keep *_mix_InitialVertices_*' ]
+_phase2_timing_extraCommands = cms.PSet( # using PSet in order to customize with Modifier
+    value = cms.vstring( 'keep *_mix_FTLBarrel_*','keep *_mix_FTLEndcap_*','keep *_mix_InitialVertices_*' )
+)
+# For premixing switch the sim digi collections to the ones including pileup
+# Unsure what to do with InitialVertices, they don't seem to be consumed downstream?
```

Comment:
@lgray @pmeridian @fabiocos @casarsa please clarify the use (if any) of `InitialVertices`